src/material-experimental/mdc-progress-bar/progress-bar.ts

```diff
@@ -50,6 +50,7 @@ export type ProgressBarMode = 'determinate' | 'indeterminate' | 'buffer' | 'quer
     'role': 'progressbar',
     'aria-valuemin': '0',
     'aria-valuemax': '100',
+    'tabindex': '-1',
```

Comment:
Could you add a comment in each of these places that gives the background for setting this?